lib/grunt.js

```diff
@@ -69,6 +69,12 @@ module.exports = function(grunt) {
 
   });
 
+  grunt.registerTask('wrapcodepoints', function() {
```

Comment:
I agree, but might be best to simply put a link to this PR (and put the SASS documentation link in the description) both here and in the `$icons` declaration.